lib/runtime/wazero/imports_test.go

```diff
@@ -569,6 +571,35 @@ func Test_ext_trie_blake2_256_root_version_1(t *testing.T) {
 	require.Equal(t, expected[:], hash)
 }
 
+func Test_ext_trie_blake2_256_root_version_2(t *testing.T) {
+	inst := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
+
+	testinput := []string{"dimartiro", "was", "here", "??"}
+	encInput, err := scale.Marshal(testinput)
+	require.NoError(t, err)
+	encInput[0] = encInput[0] >> 1
```

Comment:
Why do you right shift the first index of `encInput`?

Reply:
Actually a very good question, I based this in the one Elizabeth made for v1. I asked Tim about this right shift a few days ago to see if he knows anything. Will investigate it more about it